buildtest/menu/build.py

```diff
@@ -38,15 +38,17 @@ def discover_buildspecs(buildspec):
     """
 
     buildspecs = []
-
-    # If no config file provided, assume discovering across buildtest/site
-    if not buildspec:
-        buildspec = BUILDSPEC_DEFAULT_PATH
+    # add default path to search path
+    search_path.append(BUILDSPEC_DEFAULT_PATH)
 
     # First try, the path is an absolute path to file or folder
     # Second try, the path can be relative to the BUILDSPEC_DEFAULT_PATH
-    elif not os.path.exists(buildspec):
-        buildspec = os.path.join(BUILDSPEC_DEFAULT_PATH, buildspec)
+    if not os.path.exists(buildspec):
```

Comment:
Why did you remove the check here if it's set to None? This is again going to trigger an error if this is the case.

Reply:
Well i don't anticipate user will pass `buildtest build` we shouldnt even allow this. They should pass with `-b` option. That's why i removed the check